app/queries/servers/user.ts

```diff
@@ -106,24 +104,17 @@ export const observeUserIsTeamAdmin = (database: Database, userId: string, teamI
     const id = `${teamId}-${userId}`;
     return database.get<TeamMembershipModel>(TEAM_MEMBERSHIP).query(
         Q.where('id', Q.eq(id)),
-    ).observe().pipe(
+    ).observeWithColumns(['scheme_admin']).pipe(
         switchMap((tm) => of$(tm.length ? tm[0].schemeAdmin : false)),
     );
 };
 
 export const observeUserIsChannelAdmin = (database: Database, userId: string, channelId: string) => {
     const id = `${channelId}-${userId}`;
-    const myChannelRoles = observeMyChannel(database, channelId).pipe(
-        switchMap((mc) => of$(mc?.roles || '')),
-        distinctUntilChanged(),
-    );
```

Comment:
I think we still need to check the roles, don't we?

Reply:
The roles was a workaround to deal with this value not being properly updated. That may also had to do with graphQL doing things wrong at that time.

I guess that 'scheme_admin' is up to date and will give you the same information as checking the roles, but 0/5.